app/views/checkingRecords.scala.html

```diff
@@ -0,0 +1,6 @@
+@()(implicit messages: Messages)
```

Comment:
Is this page being used? I don't think there is a redirect to it once the upload has completed? How does this relate to the changes Loki is making in PR: https://github.com/nationalarchives/tdr-transfer-frontend/pull/84/files. I think there is a bit of overlap. Maybe this should be removed from your PR?